tests/columns/test_sql_schema.py

```diff
@@ -100,6 +103,51 @@ def test_postgres_datatype(column: Column, datatype: str) -> None:
     assert columns[0].type.compile(dialect) == datatype
 
 
+@pytest.mark.parametrize(
+    ("column", "datatype"),
+    [
+        (dy.Bool(), "SMALLINT"),
+        (dy.Date(), "DATE"),
+        (dy.Datetime(), "TIMESTAMP"),
+        (dy.Datetime(time_zone="Etc/UTC"), "TIMESTAMP WITH TIME ZONE"),
+        (dy.Time(), "TIME"),
+        (dy.Duration(), "INTERVAL DAY TO SECOND"),
+        (dy.Decimal(), "NUMERIC"),
+        (dy.Decimal(12), "NUMERIC(12, 0)"),
+        (dy.Decimal(None, 8), "NUMERIC(38, 8)"),
+        (dy.Decimal(6, 2), "NUMERIC(6, 2)"),
+        (dy.Float(), "FLOAT"),
+        (dy.Float32(), "REAL"),
+        (dy.Float64(), "FLOAT"),
+        (dy.Integer(), "INTEGER"),
+        (dy.Int8(), "SMALLINT"),
+        (dy.Int16(), "SMALLINT"),
+        (dy.Int32(), "INTEGER"),
+        (dy.Int64(), "NUMBER(19)"),
+        (dy.UInt8(), "SMALLINT"),
+        (dy.UInt16(), "INTEGER"),
+        (dy.UInt32(), "NUMBER(19)"),
+        (dy.UInt64(), "NUMBER(19)"),
+        (dy.String(), "VARCHAR2"),
+        (dy.String(min_length=3), "VARCHAR2"),
+        (dy.String(max_length=5), "VARCHAR2(5 CHAR)"),
+        (dy.String(min_length=3, max_length=5), "VARCHAR2(5 CHAR)"),
+        (dy.String(min_length=5, max_length=5), "CHAR(5)"),
+        (dy.String(regex="[abc]de"), "VARCHAR2"),
+        (dy.String(regex="^[abc]d$"), "CHAR(2)"),
+        (dy.String(regex="^[abc]{1,3}d$"), "VARCHAR2(4 CHAR)"),
+        (dy.Enum(["foo", "bar"]), "CHAR(3)"),
+        (dy.Enum(["a", "abc"]), "VARCHAR2(3 CHAR)"),
+    ],
+)
+def test_oracle_datatype(column: Column, datatype: str) -> None:
```

Comment:
I feel like we should ideally add the Oracle stuff in a separate PR since we did not support it yet (and then properly support it for all types). Wdyt @AndreasAlbertQC?

Reply:
good suggestion

Reply:
Backed those changes out. I don't need Oracle support at all, I was being overzealous trying to support DBs that have timezone support